src/gui/dialogs/campaign_selection.hpp

```diff
@@ -131,6 +131,8 @@ class campaign_selection : public modal_dialog
 	bool currently_sorted_asc_;
 
 	std::vector<std::string> last_search_words_;
+
+	inline const static std::string missing_campaign_ = "missing-campaign";
```

Comment:
Better would be something that is otherwise invalid.

Reply:
Updated.